docs/validators/import-db.md

```diff
@@ -0,0 +1,46 @@
+---
+id: import-db
+title: Import DB
+---
+
+The node is able to reprocess a previously produced database by providing the database and starting 
+the node with several flags explained in the section below. <br>
```

Comment:
please add use cases examples like:
- index in ElasticSearch (or something similar) all the data from genesis to present time
- validate the blockchain state
- make sure there aren't backwards compatibility issues with a new software version
- check the blockchain state at a specified time (this includes additional code changes, but for example if you are interested in the result of an API endpoint at the block 255255, you could use import db and force the node to stop at the block corresponding to that date)
- and so on

Reply:
done